xrootdlib/examples/XrdXrootdMon/pretty_print.py

```diff
@@ -75,7 +76,8 @@ def print_packet(initial=1):
         b'x': 'XfrInfo',
         b'r': 'Burr',
         b't': 'Buff',
-        b'f': 'Fstat'
+        b'f': 'Fstat',
+        b'g': 'Plugin'
```

Comment:
Please add a trailing comma here. This keeps the diff clean next time a new field is added.
```suggestion
        b'g': 'Plugin',
```